packages/eas-cli/src/project/customBuildConfig.ts

```diff
@@ -68,3 +69,21 @@ export async function validateCustomBuildConfigAsync({
 export function getCustomBuildConfigPath(configFilename: string): string {
   return path.join('.eas/build', configFilename);
 }
+
+export function getCustomBuildConfigPathForJob({
```

Comment:
With a separate function we could even do:
```
getCustomBuildConfigPathForJob {
  path.posix.join(...)
}

getCustomBuildConfigPath {
  path.join(...)
}
```

Reply:
you are right! thanks!